res/TensorFlowPythonExamples/examples/unique/__init__.py

```diff
@@ -0,0 +1,5 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="unique_in")
```

Comment:
other examples use input name `Hole`. please use `Hole`

Reply:
renamed to "Hole".